engine/jsb-webview.js

```diff
@@ -269,6 +269,9 @@
         this._h = node._contentSize.height;
 
         let camera = cc.Camera.findCamera(node)._camera;
+        if (!camera) {
+            return;
```

Comment:
Should validate camera at the beginning, just after line 253

Reply:
updated